### DISCUSSION TITLE
Issues with HostName Not Verified For LAN Only Bitwarden

### DISCUSSION THREAD

**Discussion by R0NAM1** [dani-garcia/vaultwarden#1600] [Q&A] [answered]
I have a bitwarden docker up on a port 7845 being forwarded, and I access it through wireguard. Issue with the SSL cert however on android 10. I have installed the cert and it recognizes it, but every time I try to connect I get the error message: Exception Message: Hostname bit.warden not verified (With SSL Cert info after, CN=Same FQDN) DNS does resolve to my server correctly, as I run my own pihole.

**Comment by R0NAM1** [comment, marked as answer]
Figured it out, when generating the OpenSSL Cert use this: `openssl req -newkey rsa:2048 -new -nodes -x509 -days 3650 -addext "subjectAltName = IP:MYIPADDRESS,DNS:MY.SITE" -keyout key.pem -out cert.pem
`

**Reply by ghost** [reply]
When you say "`MYIPADDRESS,DNS:MY.SITE`" do you mean wan ip?

**Reply by Neurology0443** [reply]
Thank you :) This worked for me !

@jeffcorona4 No, your nextcloud IP address, maybe you already figured it out :)
